### ISSUE TITLE
Expose Document event

### ISSUE THREAD

**Issue by jirisykora83** [BlazorExtensions/Home#2]

**Comment by galvesribeiro** [comment]
Sorry @jirisykora83, I don't quite understood your request.

Can you elaborate more on that?

Thank you.

**Comment by jirisykora83** [comment]
@galvesribeiro Yes of course. Sorry i publish the question before i actually finished :)

**Comment by jirisykora83** [comment]
Also i am not user if it this function is fit to BlazorExtensions or it will be better have it build-in in Blazor what do you think?

**Comment by galvesribeiro** [comment]
@jirisykora83 sorry for taking so long to reply.

Maybe we can start a project for the Document APIs here. Do you want to implement it yourself?

I'm a really _busy_ days recently and I'm not able to commit to implement it really soon.

The next thing on my queue is the SignalR client. If you want to commit to it, I can create the repo for you.

**Comment by galvesribeiro** [comment]
@jirisykora83 just to let you know. Two major perf improvements on Mono were merged Today that will directly affect blazor performance. 

Stay tunned.
